TicketDesk/ngWebClient-StandAlone/src/app/ticket-list/ticket-list.component.ts

```diff
@@ -11,21 +11,27 @@ import { FormsModule } from '@angular/forms';
 export class TicketListComponent implements OnInit {
   // This will become input
   private headingsList: string[] = ['Title', 'Status', 'Priority', 'Owner', 'Assigned', 'Category', 'Created Date'];
+  // Adds a vairable to add keep track of checkbox
+  private displayList: {'ticket': Ticket, 'checked': boolean}[];
   @Input() ticketListResults: { 'ticketList': Ticket[], 'maxPages': number };
   @Input() columns: string[];
   currentPage: number;
   ngOnInit() {
-    // State is created from ngModal in Angular
-    this.ticketListResults.ticketList.forEach(x => x.state = false);
+    this.displayList = [];
+    for (const ticket of this.ticketListResults.ticketList) {
+      if (ticket) { // ticket is of type ticket
```

Comment:
that isn't what this checks, is it? seems like some kind of filter then mapping an arrow function would work better here. not worth changing for the review, though.